runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -280,6 +289,25 @@ public void onConfigChanged(final ConfigChanged event)
 					panel.setFilterMode(config.subscriptionFilter());
 					updateList();
 					break;
+				case "customWorldCycle":
+					setCustomWorldCycle(event.getNewValue());
+					break;
+			}
+		}
+	}
+
+	private void setCustomWorldCycle(String worldList)
+	{
+		WorldResult worldResult = worldService.getWorlds();
+		worldCycleList.clear();
+		if (worldResult != null || worldList.isEmpty())
```

Comment:
`worldList` being empty should result in some kind of guard clause instead this obtuse way of setting `worldCycleList` to an empty list by parsing an empty string to a list to be added.

Reply:
Fixed, this was supposed to be an AND operator checking both was said guard.